### ISSUE TITLE
Flickr support

### ISSUE THREAD

**Issue by MaluNoPeleke** [pavelk2/social-feed#196]
Is it possible to use Flickr with social-feed?

**Comment by HartLarsson** [comment]
yes, flickr has a rss feed. You can find it by clicking more button and edit and moving to the bottom of the page you will find the rss link.

**Comment by MaluNoPeleke** [comment]
Thanks, that works but without author/profile picture and only with medium (__m.jpg) sized preview image instead of big (__b.jpg).

**Comment by HartLarsson** [comment]
Yes because flickr RSS feed proved _m size pic as a normal RSS feed. If you need to get  the _b version of the pic we need to create a special modification only for flickr feed.

if you use chrome and press f12 and look the network tab while the page is loading ans search the YQL call you will see something like this:

![image](https://cloud.githubusercontent.com/assets/16102074/17005354/100e180e-4eda-11e6-9d3b-188ecafb06e1.png)

content : is the actual part of HTML where my script grab the image _m
link:1 is instead thwe right place where grab the _b
To do that need an heavy modification of the basic js core creating a specific feed for exaple 'Flickr' liek there is 'rss' or 'facebook' with all tweaks to get that image.
Anyway the profile pic, i was not able to discover where is saved in the json answer:

And keep in mind you still using my render version, because the default js will not get any image from any RSS feed.

**Comment by MaluNoPeleke** [comment]
Then this ticket is a feature suggest ticket for Flickr support ;)
